build.gradle

```diff
@@ -1,5 +1,5 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id 'com.android.application' version '7.4.0' apply false
-    id 'com.android.library' version '7.4.0' apply false
+    id 'com.android.application' version '8.0.2' apply false
```

Comment:
Try not to change the android application version when you upload the changes in the top level gradle configuration